next.config.js

```diff
@@ -3,5 +3,14 @@ const config = withImages();
 
 config.reactStrictMode = true;
 config.poweredByHeader = false;
+config.redirects = async () => {
```

Comment:
Nie lepszy byłby tutaj rewrite? 🤔

Reply:
Co to znaczy rewrite?

Reply:
https://nextjs.org/docs/api-reference/next.config.js/rewrites